packages/eslint-plugin-gestalt/src/prefer-box-as-tag.js

```diff
@@ -0,0 +1,99 @@
+// @flow strict
+import { getHtmlTag, hasImport } from './eslintASTHelpers.js';
+import { renameTagFixer, updateGestaltImportFixer } from './eslintASTFixers.js';
+/**
+ * @fileoverview Prefer Box: prevent HTML tags supported in Box through the `as` prop
+ * @author Alberto Carreras <acarreras@pinterest.com>
```

Comment:
Up for discussion but I think we should get rid of `author` from our ESLint rules:

* We don't do it for our other files (e.g. components)
* We can get that information by looking at the git history of a file.

Reply:
DONE